Makefile

```diff
@@ -11,6 +11,10 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
+# Build settings.
+GOARCH ?= amd64
+COMPILER ?= x86_64-w64-mingw32-gcc # Cross-compiler for Windows
+
```

Comment:
So by default it would take the assigned values but we can override them during make command invocation.
For example-
`make release` would lead to below command-
```
GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc go build -buildmode c-shared -o ./bin/firehose.dll ./
```

However if we use `GOARCH=arm64 make release` would invoke-
```
GOOS=windows GOARCH=arm64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc go build -buildmode c-shared -o ./bin/firehose.dll ./
```